### ISSUE TITLE
Command bwaddon does not exist

### ISSUE THREAD

**Issue by Maks1116** [pronze/SBAHypixelify#110]
When I try to make npc I need to provide command, that executes bwaddon, but this command does not exist

**Comment by Maks1116** [comment]
Oh, and also villagers don't exist. I should have SBAHypixelify folder in plugins, but I don't have it

**Comment by VoidyCD** [comment]
same issue here

**Comment by lolo45654** [comment]
same

**Comment by VoidyCD** [comment]
i know the fix
download a version before .4

**Comment by lolo45654** [comment]
before 1.4.0?

**Comment by VoidyCD** [comment]
yes version 1.3.9 should work. and make sure you have java 11 or higher on
your server.

**Comment by lolo45654** [comment]
when i type java -version it says that java 16 is installed

**Comment by lolo45654** [comment]
thank you so much!

**Comment by VoidyCD** [comment]
not your java version. the server's java version

**Comment by lolo45654** [comment]
i dont know what you mean but it works
thanks
